src/Yarp.ReverseProxy.Swagger/ReverseProxyDocumentFilter.cs

```diff
@@ -163,6 +170,11 @@ DocumentFilterContext context
                                     }
                                 }
 
+                                foreach (var operationKey in operationKeys)
+                                {
+                                    ApplySwaggerTransformation(operationKeys, path, routes);
```

Comment:
You use `operationKeys` but not `operationKey`. `foreach` is not necessary in this case.

Reply:
Correct. I did some refactoring before creating the pr. I missed that bit. Will fix when I'm back at the PC.